reconfiguration/include/reconfiguration/ireconfiguration.hpp

```diff
@@ -27,51 +27,43 @@ class IReconfigurationHandler {
   // Message handlers
```

Comment:
In order not to have trivial implementations in subclasses, since not all the handlers handle all commands, I would not make these methods pure virtual, but rather added a trivial impl `{return true;}`

Reply:
fixed